src/main/java/edu/umich/tl/ApiCallHandler.java

```diff
@@ -0,0 +1,82 @@
+package edu.umich.tl;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.http.HttpResponse;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.message.BasicNameValuePair;
+
+public class ApiCallHandler {
+	private static Log M_log = LogFactory.getLog(ApiCallHandler.class);
+	private String canvasURL=CourseDelete.canvasURL;
+	private String esbURL=CourseDelete.esbURL;
+	private static String canvasToken=CourseDelete.canvasToken;
+	
+
+	public enum ApiTypes{
+		Term(1), UnPublishedCourseList(2);
+		private int apiType;
+		private ApiTypes(int apiType) {
+			this.apiType=apiType;
+		}
+		public int getApiType() {
+			return apiType;
+		}
+	}
+	public HttpResponse getApiResponse(ApiTypes apiType, int typeOfApiCall) {
```

Comment:
Change typeOfAPiCall to a boolean (e.g. useESB) or an enum (e.g. DIRECT_CANVAS, ESB_CANVAS); Change apiType to canvasApi -- both these two variables are too similar and poorly named (think self-documenting code -- good variable names are just as important as good comments)